### PR TITLE
[#6352] Skip msiExecCmd_bin when chown-ing directories

### PR DESCRIPTION
**Pull request by SwooshyCueb**

### CODE REVIEW

File: scripts/chown_directories_for_postinstall.py [hunk 1]
```diff
@@ -6,5 +6,10 @@
 if os.path.exists(paths.service_account_file_path()):
     service_account_uid = paths.irods_uid()
     service_account_gid = paths.irods_gid()
-    for dirpath, _, _ in os.walk(paths.irods_directory()):
+    is_top = True
+    for dirpath, dirnames, _ in os.walk(paths.irods_directory()):
+        if is_top:
```
Comment (alanking) [inline scripts/chown_directories_for_postinstall.py:11]: Just for my own understanding, this check is here to ensure that we are only skipping the "official" msiExecCmd_bin and not one that may be deeper down and was added by the administrators for whatever reason?

**Review by korydraughn — APPROVED** [review]

File: scripts/chown_directories_for_postinstall.py (hunk 1 above)
Comment (SwooshyCueb) [reply scripts/chown_directories_for_postinstall.py:11]: Correct

Comment (alanking) [reply scripts/chown_directories_for_postinstall.py:11]: Cool, thanks!

**Review by alanking — APPROVED** [review]

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**